sources/_sector/dex/trades/ethereum/_sources.yml

```diff
@@ -113,6 +113,10 @@ sources:
     tables:
       - name: XchangePair_evt_Swap
       - name: XchangeFactory_evt_PairCreated
+  - name: xchange_v2_ethereum
+    tables:
+      - name: XchangePair_evt_Swap
+      - name: XchangeFactory_evt_PairCreated
```

Comment:
These sources look very similar...
We have two uniswap v2 forks xhange here.

Reply:
Seems they are different contracts:
[Xchange intersection test](https://dune.com/queries/4188446)